frontend-instructor/src/components/signup/SignUp.jsx

```
@@ -0,0 +1,179 @@
+            <input
+              type="text"
+              name="name"
+              className="w-full p-2 border rounded-md"
+              onChange={signupFormik.handleChange}
+              value={signupFormik.values.name}
+              required
+            />
+          </div>
+
+          <div>
+            <label className="block text-sm font-medium">Email</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)